mysite/app/templates/app/tilbud_liste_manager.html

```diff
@@ -7,31 +7,36 @@
 
 {% block content %}
 <div class="page-container">
-    <h1>Oversikt over tilbud:</h1>
-    <h3>Du har {{ antall_tilbud }} tilbud i innboksen din</h3>
-    <div class="page-content">
-        {% if tilbuds %}
-            <table>
-                <thead>
+    <div class="space_around">
+        <h1>Oversikt over tilbud:</h1>
+    </div>
+    <div class="space_around">
+        <h3>Du har {{ antall_tilbud }} tilbud i innboksen din</h3>
+    </div>
+    <div id="overview" class="page-content">
+        {% if antall_tilbud > 0 %}
```

Comment:
Denne bruker du i stedet for de to id-ene og alt innenfor denne skal vel ha en indent?

Her kan du lære av hva jeg har gjort på det andre siden jeg tror at du skal kunne skrive:
`if tilbud.artist.all`